aep/general/0159/aep.md.j2

```diff
@@ -1,5 +1,74 @@
 # Reading across collections
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/13) for more
-information.
+Sometimes, it is useful for a user to be able to retrieve resources across
+multiple collections, or retrieve a single resource without needing to know
+what collection it is in.
+
+## Guidance
+
+APIs **may** support reading resources across multiple collections by allowing
+users to specify a `-` (the hyphen or dash character) as a wildcard character
+in a standard [`List`](./list) method:
+
+```
+GET /v1/publishers/-/books?filter=...
+```
+
+- The method **must** explicitly document that this behavior is supported.
+- The resources provided in the response **must** use the canonical name of the
+  resource, with the actual parent collection identifiers (instead of `-`).
+- Services **may** support reading across collections on `List` requests
+  regardless of whether the identifiers of the child resources are guaranteed
+  to be unique. However, services **must not** support reading across
+  collections on `Get` requests if the child resources might have a collision.
+- Cross-parent requests **should not** support `order_by`. If they do, the
+  field **must** document that it is best effort. This is because cross-parent
+  requests introduce ambiguity around ordering, especially if there is
+  difficulty reaching a parent (see [unreachable resources]).
+
+**Important:** If listing across multiple collections introduces the
+possibility of partial failures due to unreachable parents (such as when
+listing across locations), the method **must** indicate this following the
+guidance in [unreachable resources].
+
+{% tab proto %}
+
+- The URI pattern in the `google.api.http` annotation **must** still be
+  specified with `*` and permit the collection to be specified; a URI pattern
+  **must not** hard-code the `-` character.
+
+{% endtabs %}
+
+### Unique resource lookup
+
+Sometimes, a resource within a sub-collection has an identifier that is unique
+across parent collections. In this case, it may be useful to allow a
+[`Get`](./get) method to retrieve that resource without knowing which parent
+collection contains it. In such cases, APIs **may** allow users to specify the
+wildcard collection ID `-` (the hyphen or dash character) to represent any
+parent collection:
+
+```
+GET https://library.exampleapis.com/v1/publishers/-/books/{book}
+```
+
+- The URI pattern **must** still be specified with `*` and permit the
+  collection to be specified; a URI pattern **must not** hard-code the `-`
+  character.
```

Comment:
Should this also be moved to a protocol buffers tab?

Reply:
Thanks for calling this out.  We discussed it live today in our regular meeting and concluded that the guidance is good, but that the phrasing should be genericized a bit (and weakened to a **should** rather than a **must** for non-gRPC APIs).  Updated.